core/src/text.rs

```diff
@@ -0,0 +1,23 @@
+use ropey::Rope;
```

Comment:
I think this file is not needed. I moved everything in `src/text.rs` to `src/prim/text.rs`